index.php

```diff
@@ -153,6 +153,12 @@
 	define('PHPBB_32', true);
 }
 
+// Check if equal or greater than 4.0@dev
```

Comment:
Comment here not really needed (we're not using them above so why start now....or otherwise add comments above)